e2e/testdata/fn-eval/function-env/.expected/setup.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# Build a docker image to print env
+IMAGE_TAG="gcr.io/kpt-fn/local-printenv:unstable"
+docker build -t $IMAGE_TAG - <<EOF
+FROM busybox:stable
+CMD ["printenv"]
+EOF
```

Comment:
hmm..why not have this image hosted in `gcr.io` ? It's better to not depend on images from `dockerhub` because we might run in to some rate-limiting issues (I remember they introduced some in recently).

Reply:
Working with @mengqiy to have a GCR for test/demo functions

Reply:
Good thing is once that image is in `gcr`, all we have to do will be just drop the `setup.sh`. You can change `gcr.io/kpt-fn` to `gcr.io/kpt-demo-fn` (whatever the correct name is).

Reply:
done